db/migrate/20201215215458_create_nod_date_edit.rb

```diff
@@ -0,0 +1,13 @@
+class CreateNodDateEdit < Caseflow::Migration
+  def change
+    create_table :nod_date_edits do |t|
```

Comment:
For consistency with tables like `request_issues_updates` and `end_product_updates`:
```suggestion
    create_table :nod_date_updates do |t|
```